.github/workflows/publish-olm-bundle.yaml

```diff
@@ -0,0 +1,58 @@
+name: Submit release OLM bundle
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  createPullRequest:
+    name: Publish new OperatorHub release
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Build the bundle
+        run: pushd deployment/operator && make bundle && popd
+
+      - name: Checkout tools repo
+        uses: actions/checkout@v4
+        with:
+          repository: k8s-operatorhub/community-operators
+          path: community-operators
+          ref: main
+          token: ${{ secrets.MACHINE_PAT }}
+
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@v6
+        id: import-gpg
+        with:
+          gpg_private_key: ${{ secrets.MACHINE_GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.MACHINE_PASSPHRASE }}
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+          workdir: community-operators
+
+      - name: Copy the bundle to the community-operators repo
+        run: |
+          mkdir -p operators/nri-plugins-operator/${{ github.ref_name }}
+          cp -r ../deployment/operator/bundle/* operators/nri-plugins-operator/${{ github.ref_name }}
```

Comment:
Can it happen, that for some reason the operator bundle will have an entirely new file which is not part of the repo yet and we'd need to add it to the repo ?

Reply:
No, if we don't bump the operator-sdk in the same PR, without testing it.

Reply:
even if it adds, I think it will always follow the same path, meaning dump everything under `bundle/ `directory.